packages/python/plotly/plotly/express/_core.py

```diff
@@ -1330,7 +1330,7 @@ def init_figure(args, subplot_type, frame_list, nrows, ncols, col_labels, row_la
                 main_size = 0.84
 
             row_heights = [main_size] * (nrows - 1) + [1 - main_size]
-            vertical_spacing = 0.01
+            vertical_spacing = 0.012 * 600 / args["height"]
```

Comment:
If we're checking for width being None below, shouldn't we check for height being None here?

Reply:
Also, instead of multiplying two constants, could you just use 7.2?

Reply:
Thanks for your comments @marthacryan . I'm a bit removed from this PR since it was a while since I created it, but your suggestions here and in the other comment sounds good so feel to make those edits (any others you see fit) directly on this branch.

Reply:
/me waves at @joelostblom

Reply:
Haha, hello Greg! Nice to see you here =)